config/backstop/backstopConfig.js

```diff
@@ -43,27 +43,27 @@ const config = {
     {
       ...basicScenario,
       label: 'h1-heading',
-      selectors: ['h1']
+      selectors: ['h1'] // only one h1 on the page
     },
     {
       ...basicScenario,
       label: 'h2-heading',
-      selectors: ['h2']
+      selectors: ['h2:nth-of-type(8n + 1)'] // check few h2
```

Comment:
why few? if styles for one h2 are broken all others will be broken too. We should aim for the minimal required number of tests/screenshots.